webpack/ForemanTasks/Components/TasksDashboard/TasksDashboardSelectors.js

```diff
@@ -12,6 +12,9 @@ export const selectTime = state =>
 
 export const selectQuery = state => selectTasksDashboard(state).query || {};
 
+export const calcStoppedOther = data =>
+  (data?.cancelled?.total || 0) + (data?.pending?.total || 0);
+
```

Comment:
Can we get this data from the server?

Reply:
I think it will complicate the server more that this complicated the front end

Reply:
commented about it also here: https://github.com/theforeman/foreman-tasks/pull/578#discussion_r477113509

Reply:
We don't need this anymore :)

Reply:
Why not?

Reply:
Don't you pass it as otherCount from server?

Reply:
Nope, the server returns {[result]:count} for every result in the DB

Reply:
can you add a small test for `calcStoppedOther`?